### PR TITLE
Initial Implementation

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+service: aqts-ts-type-router
+
+provider:
+  name: aws
+  region: ${opt:region, 'us-west-2'}
+  stage: ${opt:stage, 'TEST'}
+  runtime: java8
```
Comment (ssoper-usgs) [inline serverless.yml]: Just confirming we want to use java8 rather than java11 here.

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://${AQTS_DATABASE_ADDRESS}:${AQTS_DATABASE_PORT}/${AQTS_DATABASE_NAME}?currentSchema=${AQTS_SCHEMA_NAME}
+    username: ${AQTS_SCHEMA_OWNER_USERNAME}
+    password: ${AQTS_SCHEMA_OWNER_PASSWORD}
+    hikari:
+      maximumPoolSize: 3
```
Comment (ssoper-usgs) [inline src/main/resources/application.yml]: This might need to be `maximum-pool-size`

File: src/test/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://${AQTS_DATABASE_ADDRESS}:${AQTS_DATABASE_PORT}/${AQTS_DATABASE_NAME}?currentSchema=${AQTS_SCHEMA_NAME}
+    username: ${AQTS_SCHEMA_OWNER_USERNAME}
+    password: ${AQTS_SCHEMA_OWNER_PASSWORD}
+    hikari:
+      maximumPoolSize: 2
```
Comment (ssoper-usgs) [inline src/test/resources/application.yml]: This might need to be `maximum-pool-size`

File: serverless.yml (hunk 1 above)
Comment (dsteinich) [reply serverless.yml]: For this go around, yes java8. java11 was causing issues and appeared to require more memory/time to load

**Pull request merged**